style.css

```diff
@@ -0,0 +1,8 @@
+#navbar{
+    float: left;
+}
```

Comment:
**Optional**

- [ ]  Very good👍 way to use id attributes, but for your code to be shorter and well structured it would be better to use class attributes to apply the style to several blocks or elements in the same page.